test/test.js

```diff
@@ -37,4 +37,14 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {
+      detectImageSize: true,
+      resolveImagePath: (value, source) => {
+        expect(typeof value).toBe('string');
+        expect(source).toHaveProperty('input.css');
+        return path.resolve(process.cwd(), value);
+      }}, done);
+  });
```

Comment:
Ah yes, this makes sense. I was wondering if async calls would be properly resolved, but I forgot that `done` callback is called "long after" `expect` calls are resolved.

I’ve updated tests with these changes.